### PR TITLE
Fix testsuite #6097

### LINKED ISSUE #6071: Config tests fails
```text
ConfigTests
    ✔ testChangeChainID (0.002 seconds)
    ✔ testMakeSureDevelopmentFlagsAreAllFalse (0.001 seconds)
    ✖ testSwitchLocale, XCTAssertEqual failed: ("nil") is not equal to ("Optional("Wallet")")
    ✖ testSwitchLocale, XCTAssertEqual failed: ("nil") is not equal to ("Optional("我的钱包")")
```

### LINKED ISSUE #6097: Make test fails with 4 failures
The following tests failed:

SendViewControllerTests.testScanEip681QrCodeEnglish()
SendViewControllerTests.testScanEip681QrCodeEnglish()
SendViewControllerTests.testScanEip681QrCodeSpanish()
TransactionTypeFromQrCodeTests.testScanSmallEthTransfer()

### LINKED ISSUE #6101: Should not be able to add a custom token with address `0x0000000000000000000000000000000000000000`
Might not because of #6096, but it's related.

1. Add / Hide Tokens etc
2. Tap Auto and change to a specific chain, e.g. Goerli
3. Paste `0x0000000000000000000000000000000000000000`
4. Symbol = `a`
5. Balance = `0`
6. Name = `a`
7. Tap Done

Expected
---
8. Some error

Observed
---
8. Operation goes through and the native token is now labelled as "a"

### PR DESCRIPTION
**Pull request by oa-s**
Fixes #6097, Closes #6101, Closes #6071

### CODE REVIEW

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/TokensAutodetector/ImportToken.swift [hunk 1]
```diff
@@ -114,6 +109,11 @@ final public class ImportToken: TokenImportable, TokenOrContractFetchable {
         }.then(on: queue, { [queue] contract -> Promise<TokenOrContract> in
             let key = "\(contract.hashValue)-\(onlyIfThereIsABalance)-\(server)"
 
+            //Useful to check because we are/might action-only TokenScripts for native crypto currency
```
Comment (hboon) [inline modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/TokensAutodetector/ImportToken.swift:112]: Works, but should change the commit message though. We fixed the code, not just the test suite

**Review by hboon — APPROVED** [review]

File: modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/TokensAutodetector/ImportToken.swift (hunk 1 above)
Comment (oa-s) [reply modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/TokensAutodetector/ImportToken.swift:140]: tests are still fail, trying to figure it out

**Comment by hboon** [comment]
Passes for me too, didn't check though

**Comment by hboon** [comment]
But remember, as it might be important to find later:

> Works, but should change the commit message though. We fixed the code, not just the test suite

**Pull request merged**